src/Control/Lens/Iso.hs

```diff
@@ -427,81 +409,25 @@ flipped :: Iso (a -> b -> c) (a' -> b' -> c') (b -> a -> c) (b' -> a' -> c')
 flipped = iso flip flip
 {-# INLINE flipped #-}
 
--- | This class provides for symmetric bifunctors.
-class Bifunctor p => Swapped p where
```

Comment:
I hadn't considered the need to migrate `Swapped`, but I guess if we're going to migrate `Strict`, then `Swapped` would also be included. On the other hand, unlike `Strict`, there are actually uses of `Swapped` in Hackage libraries "in the wild" (a quick search reveals `accuerr`, `pipes-key-value-csv`, `validation`, and `separated`). As a result, please document carefully how to migrate from `lens:Swapped` to `assoc:Swap` in the `CHANGELOG` (in addition to how to migrate from `lens:Strict` to `strict:Strict`).

Reply:
It's not strictly necessary. If `lens` depends on `strict`, than it makes sense to move all instances from `strict-lens` to `lens`.

`Swapped` is one. Removing whole class is easier approach :)

That reminds me, that I have to add `Each` instances for `strict` types to `lens`.